python/spacewalk/server/dbConfig.py

```diff
@@ -0,0 +1,64 @@
+#  pylint: disable=missing-module-docstring,invalid-name
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025 SUSE LLC
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+from . import rhnSQL
+
+
+def value(key):
+    """
+    Return the value of the given key. If the value is not defined, return the default value
+    When the key is not found, an AttributeError is raised
+    """
+
+    def sanitize_value(key, val):
+        """
+        attempt to convert a string value to the proper type
+        """
+        convert_table = {
+            "PSW_CHECK_SPECIAL_CHARACTERS": str,
+        }
+        val = val.strip()
+
+        if convert_table.get(key):
+            try:
+                val = convert_table.get(key)(val)
```

Comment:
This will raise a TypeError when `dict.get()` returns it's default value `None`.

Reply:
I think we should be good here, as we are inside the `if convert_table.get(key)` statement, so value shouldn't be `None`.